src/configParser.js

```diff
@@ -76,6 +79,24 @@ ConfigParser.prototype._mergeConfig = function (configFile, type) {
   this.config = _mergeWithArrays(newConfig, priorityConfig);
 };
 
+ConfigParser.prototype._resolveProfile = function (profile, fullConfig) {
+  fullConfig = fullConfig || [];
+  if (!profile) {
+    return [];
+  }
+  var builtInFilePath = path.join(CONFIG_DIR, profile + '.profile.conf.js');
+  var pathToConfig = profile.indexOf(path.sep) > -1 ? profile : builtInFilePath;
```

Comment:
this is a tricky choice - what if the user gives the simplest form of a relative path without separator - like "own.profile.conf.js"?. IMHO the readSync was good idea, we just needs a solution for the cwd vs installed dir problem. I was thinking of solving this by simply prepending the default files with __dirname and adding path.resolve(). This way require() will always get absolute path and no ambiguity will arise ?